CONTRIBUTING.md

```diff
@@ -2,68 +2,68 @@
 
-Reductions require an estimator to be passed that implements the `fit` method with the `sample_weight` argument. The constraints for reductions are all moments (see `fairlearn.reductions`) passed as instances of classes inheriting from `Moment`. Moments are vector functions that we use to formalize our constraints. The moment objects need to be passed as constraints to the constructor of a reduction technique, which internally uses the constraints to solve the optimization problem.
+#### Reductions
+
+Reduction constructors require a parameter corresponding to an estimator that implements the `fit` method with the `sample_weight` argument. Parity constraints for reductions are expressed via instances of various subclasses of the class `fairlearn.reductions.Moment`. Formally, instances of the class `Moment` implement vector-valued random variables whose sample averages over the data are required to be bounded (above and/or below).
 
 ```python
 constraints = Moment()
-reduction = Reduction(estimator, objective=objective, constraints=constraints, **kwargs)
+reduction = Reduction(estimator, constraints)
 ```
 
-Reduction-based disparity mitigation algorithms (such as the ones under `fairlearn.reductions`) provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
+Reductions  provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
 
 ```python
-reduction.fit(X, Y, sensitive_features=sensitive_features)
+reduction.fit(X, y, **kwargs)
 reduction.predict(X)
 reduction._pmf_predict(X)
 ```
 
-where `sensitive_features` contains data on which group a sample belongs to. As of now, sensitive features can only be provided through `sensitive_features`. In the future we plan to allow specifying specific columns of `X` as sensitive features, in which case `sensitive_features` would be optional.
+All of the currently supported parity constraints (subclasses of `Moment`) are based on sensitive features that need to be provided to `fit` as a keyword argument `sensitive_features`. In the future, we will also allow providing sensitive features as columns of `X`.
 
-### Post-processing methods
+#### Post-processing algorithms
 
-Post-processing methods require an already trained predictor. For consistency we also provide the option to pass an estimator instead, and will call `fit` internally. For post-processing methods we provide the `constraints` argument in the form of a string.
+The constructors of post-processing algorithms require an already trained predictor as an argument. For consistency, we also allow providing an estimator instead, and the estimator is then fitted on the data at the beginning of the execution of `fit`. For post-processing algorithms, the `constraints` argument is provided as a string.
```

Comment:
I'm not fond of "allow providing" as a phrase.

Reply:
rephrased